pdf-aam/index.html

```diff
@@ -191,33 +141,40 @@ <h2>General rules</h2>
 
         <section id="mapping_role_table">
           <h3>Role Mapping Tables</h3>
-          <h4 id="role-map-example"><code>example</code></h4>
+          <h4 id="role-map-example"><code>Document Logical document within a parent document</code></h4>
           <table class="data" aria-labelledby="role-map-example">
             <tbody>
```

Comment:
And then the aria-labelledby reference here should be updated to the id  of the h4.